createdisk.sh

```diff
@@ -334,7 +334,7 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl enable io.podman
 
 # Remove all the pods except openshift-sdn from the VM
 pods=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo crictl pods -o json | jq '.items[] | select(.metadata.namespace != "openshift-sdn")' | jq -r .id)
-${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- "sudo crictl stopp ${pods} || true"
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- "sudo crictl stopp '${pods}' || true"
```

Comment:
Looks like this doesn't fix the issue since now it will take the whole part as a single pod. we need to use the double quotes like we are using in L#338.

```
$ ssh crc -- "sudo crictl stopp '${pods}'"
Warning: Permanently added 'api.crc.testing,192.168.130.11' (ECDSA) to the list of known hosts.
Stopped sandbox 4abe6678234de0bc5dcd49494fd47bec29114f3f0b3d160bafccf3bcf3f3d305
2718962d6ccbe61561e3683eea7fbf3dafbfeab0bb6b795aed98f765bd289aa9
a7a7eb99267e080d420453114eb451508b2df2a2f5903730845246987e311ab8
c3212f1b6e9d1de2cd34c4e76156cd8e33d1a8d31229f530ae9641a4fa9c73f5
9a1bdde102f99c9d6af0c1e3ed4f80970649116bd334518b4a2efd92ddc0ef31
821260535e6353ef9b61f4fd1370e49e6bd2fa70e3b9355303a634381cedcf5a
ee035db9334a17d749594b91cb0057a209dbb5cdf538acc69919c6dab3f06afc
aac9de47188e48066fbb9ef8d04360f19050d0be49783d5b95b0c4bce8ecc2cc
a43f500a4f753ac6a43092585ca26a0fb2a97f97e5a2e22e3e4cf44b7cd01b00
04a239d9f45b175451c96e5d2f1fa66bad7175a9af050a9986f751f0e8bac028
1d4e59fdb00bd10b6c9614419b18be92b0949d0e0cabed0fab387da51adf8e16
4f0dfe39b3ef73364315c734c2a9a1291e9c909c7e2a05778c3782d8e4838560
1f525908c1027b37a80dfc65087f3ee625dc1192e68b5ea964b96439604e7e62
27eeee991f5830082b386044bb8840c52267197a200bacd504b72d6c3b8941de
a15861172b79908817265bf6b8e9c6ca65c672002022c358e41e4681fc13eeef
5ae6e8567d819079bace24b45b4ac133a93af5fe4587be8b9707845ec8f503d4
92afa6a89c926f1d54fe94c64d7139bbcc4f811cc4a1b55204454372af60499c
3418eb61492c033b9a83baf28f112cb88a391cd710ca294e7d2df394e8b94e76
3c6a01432808e2eb3c79d9791e88cccc06018279882ac7fc84ee29bf6a8ae7b0
d93e5114285624bdc17175b5931d0e5e4db3f0f79bf61dae0b861cf2f0311d4a
5679f32f14fff2f861396a8837b6738108fbbf4e469a3dacd96687964fb9f0ba
f719b07bf9a47e186dc4d5cc88c5382f475667134c5a66c13f16bac0c784bb72
1f1e720c431d54139951c3c0b9bdbb8e33bacdff708388afff571b1fb0433174
c7dd5c76c88fdeb24d2f3ecf81a96bbe8ab6f418b384b27ec2b7ac9af1c615a8
4dc0976dbde31a15c1eed2c9bd4180fb009dc1c6bbbcfaf8ad14bdc63ceaea3b
f3474201f0ce3a17062c75508829720123efd81b29632e7a76fa725585191976
7d9366019143d53c1c4a2a7b21113024d186155d68134f9ffa81ba1089938c38
2b828e35dc6b5e9df09cd718520f6c77182bcb1d4b1843c73e0c9e8212d7f1f9
cef985be855004381f5ad2cef6dfd9a977b9459b8c689f403bcb0713906dbe40
e03ba7b1d974dea489f4a997ac051974bf734922cae673c8414bf2b96482aaa7
68b01d4068dfc4fbaa45d2ef1645d062fb1eb963560205dffa1b969a39c97298
1a6425ff160a1961d8bd2d59f3db48eb8319f0a4ae281caf8bd94b85951369ca
85d5ceebd1b9605d1fd62f25a528b3a63e0f21d331ac1603715df9f541527eb1
165d031fc2fa9c9ed38a1f93953d404d9fb76593d58705592e7cdf20ef62ff5e
8d31430ab9f0f1f5bba09535338a3fd9fbeb6f29cabfcd216d6e9b5bc1b10c7e
8bcbcb69df3278c356d9c1d0de0c8742d6bdcb53c679002f4a78bfa539b652eb
5cd864690a2b7dc160880719400d04468a84cd69a1325f39fa35563a0dd09767
4babb00f7cb151afe106b60034542ae46d4b55301ae9921ecd615e0e90023b6f
2e1f5c05c88700f9e95eb43cc8d31f7188b6f2ba02ef7f4cfa84d859779c3f28
33fa712abc746bbdee8eb3d5b3ec4dca9658782b257f35f5a62e966d48843125
be6a18f071a41376d830ca34cfd11006fff066e0249bc138258fc1e7d96bb530
0639256fe28c4c3f2a2c782fbbdc1846c843f764f7f51919a4635f33d0c727a4
44099f92e9444f39232fecf73071d181aa9738708a7a17a176d7933a43f94601
8018f96e5af53087cb98bbde5b31b88373ccc4a2de484060f8abbed9331e26b7
dc8b6c0bdba9a7701d34ac3320c6758bbbd5222be3cf506f490f43f71bd529f7
a9ccc0eab480108cdbf9b2bd2d9582702463bcf4777de09b01d91dc0c5dddc22
98c8f10dbd90594390076ceede1d4d181607adf72abfdfc8cef5d14d20ca6090
cec1700ae588c76818eb8755e7d91cbb1ec4fb4dd19ff777d0e07fa39ce4ae69
65e620fa35549a4c4115828a264cfbb89d67996a02ce515991e449009abfc71a
657c2590a41a19df995db5faed082615e586ba6818b19f2bb9621440b54f19e8
7c337bcf04a8e2d8abacf8ce5c7d371dd7a70dd10658a0243a9e999b8cd750bf
fbf56ca568dd2cbb45f4ebac13a3d44d1612a31c44c084588cb065b48e9f32ce
88689cf2c9475537a09f833f7702392d0bbb975ac19f27eb348ac04248a7f511
15848199581f3d815962007d6b2842621be6e8413e0454a60f81b36ad247b23d
2ac8cfb506d87b08adb6628bbf320ea8feb506f13657152deaf98d5ccae632fe

$ ssh crc -- "sudo crictl stopp "${pods}""
Warning: Permanently added 'api.crc.testing,192.168.130.11' (ECDSA) to the list of known hosts.
Stopped sandbox 4abe6678234de0bc5dcd49494fd47bec29114f3f0b3d160bafccf3bcf3f3d305
Stopped sandbox 2718962d6ccbe61561e3683eea7fbf3dafbfeab0bb6b795aed98f765bd289aa9
Stopped sandbox a7a7eb99267e080d420453114eb451508b2df2a2f5903730845246987e311ab8
Stopped sandbox c3212f1b6e9d1de2cd34c4e76156cd8e33d1a8d31229f530ae9641a4fa9c73f5
Stopped sandbox 9a1bdde102f99c9d6af0c1e3ed4f80970649116bd334518b4a2efd92ddc0ef31
Stopped sandbox 821260535e6353ef9b61f4fd1370e49e6bd2fa70e3b9355303a634381cedcf5a
Stopped sandbox ee035db9334a17d749594b91cb0057a209dbb5cdf538acc69919c6dab3f06afc
Stopped sandbox aac9de47188e48066fbb9ef8d04360f19050d0be49783d5b95b0c4bce8ecc2cc
Stopped sandbox a43f500a4f753ac6a43092585ca26a0fb2a97f97e5a2e22e3e4cf44b7cd01b00
Stopped sandbox 04a239d9f45b175451c96e5d2f1fa66bad7175a9af050a9986f751f0e8bac028
Stopped sandbox 1d4e59fdb00bd10b6c9614419b18be92b0949d0e0cabed0fab387da51adf8e16
Stopped sandbox 4f0dfe39b3ef73364315c734c2a9a1291e9c909c7e2a05778c3782d8e4838560
Stopped sandbox 1f525908c1027b37a80dfc65087f3ee625dc1192e68b5ea964b96439604e7e62
Stopped sandbox 27eeee991f5830082b386044bb8840c52267197a200bacd504b72d6c3b8941de
Stopped sandbox a15861172b79908817265bf6b8e9c6ca65c672002022c358e41e4681fc13eeef
Stopped sandbox 5ae6e8567d819079bace24b45b4ac133a93af5fe4587be8b9707845ec8f503d4
```